app/src/main/java/ch/epfl/culturequest/database/Database.java

```diff
@@ -0,0 +1,23 @@
+package ch.epfl.culturequest.database;
+
+import java.util.concurrent.CompletableFuture;
+
+public class Database  implements DatabaseInterface{
+    private static DatabaseInterface  databaseInstance = new FireDatabase();
+
+    public static void init(DatabaseInterface database) {
```

Comment:
What is this method used for? 
Should it be called before using `set` and `get`?

Reply:
This is useful for testing purpose because you can call Database.init(New MockDatabase()) an it will replace the database everywhere in the code with a MockDatabase.
I added comments about it

Reply:
ahh okok i see, thanks for clearing this up